src/Genre/Genre.tsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+import React from 'react';
 
-function Genre() {
-  return (
-    <section className='genre'>
-      <h1>Genre</h1>
-    </section>
-  )
+interface IProps {
+	updateSelectedGenre: (genre: string) => void;
+	genre: string;
 }
 
-export default Genre;
\ No newline at end of file
+function Genre(props: IProps) {
+	return (
+		<section className='genre'>
+			<h1
+				onClick={event =>
+					props.updateSelectedGenre(event.currentTarget.innerHTML)
```

Comment:
innerText was returning `undefined` so we switched to innerHTML. Looks good for now, but I'd like to know the root of this issue.